test/parallel/test-process-startTime.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const now = Date.now();
+const diffFromNow = now - process.startTime;
+
+// Make it a bit flexible
+assert.ok(diffFromNow > 0);
+assert.ok(diffFromNow < common.platformTimeout(100));
```

Comment:
Does that mean you suggest to remove both assertions? 😆 

I suppose I could do a `Number.isInteger` and/or `Number.isFinite` on it as a test instead?